lib/hammer_cli_foreman/smart_variable.rb

```diff
@@ -139,9 +138,13 @@ class AddOverrideValueCommand < HammerCLIForeman::CreateCommand
       end
     end
 
-    class RemoveOverrideValueCommand < HammerCLIForeman::DeleteCommand
+    class AddOverrideValueCommand < AddMatcherCommand
```

Comment:
Please use the same approach as you did in smart class params (ie. hiding the command and add a warning). We should be consistent.